rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/ApplicationResource.java

```diff
@@ -620,11 +620,21 @@ private void checkLocationsAreValid(ApplicationSpec applicationSpec) {
 
     @Override
     public List<EntitySummary> getDescendants(String application, String typeRegex) {
+        Entity entity =brooklyn().getApplication(application);
+        if (entity != null && !Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ENTITY, entity)) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to see the entity '%s'",
+                    Entitlements.getEntitlementContext().user(), entity);
```

Comment:
Message should say `User '%s' is not authorized to see the descendants of entity '%s'"`
Please also check the parameters for the templated message

Reply:
I think you missed this one